src/MxcImageProvider.cpp

```diff
@@ -21,6 +22,8 @@
 #include "Utils.h"
 
 QHash<QString, mtx::crypto::EncryptedFile> infos;
+// bit-shift for 2 MB makes me feel like a 1337 H4XXOR somehow
+QCache<QString, QImage> cachedThumbnails{2 << 20};
```

Comment:
Well, you could have just written 2 * 1024 * 1024 ;p

Reply:
What's the fun of that? :laughing:

Reply:
(Would you prefer that?)